debian/copyright

```diff
@@ -1,7 +1,7 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-name: bmap-tools
-Upstream-Contact: Artem Bityutskiy <artem.bityutskiy@linux.intel.com>
-Source: https://github.com/01org/bmap-tools
+Upstream-name: bmaptool
+Upstream-Contact: Trevor Woerner <twoerner@gmail.com>
```

Comment:
Are you a Debian packager? can you actually push to debian?

Reply:
No I am not, I was just doing a blanket search-and-replace on a string.

I 100% want the code repository for this piece of software to not carry any of the other strings ever again. Do we have to carry rpm-specific and debian-specific packaging information in the source code repository for a specific app? Shouldn't debian or redhat be responsible for their own packaging information? It's a bit messy to push all of this out to the software repositories themselves, no? Where does it stop? Should we also carry the Yocto recipe here too? ;-)

Reply:
I think it's fine for now. We'll let upstream packagers for those distributions do their own thing.